.github/CODEOWNERS

```diff
@@ -1,2 +1,5 @@
-# This should match the owning team set up in https://github.com/orgs/opensearch-project/teams
-*   @opensearch-project/logstash-input-opensearch
\ No newline at end of file
+# This should match the list of maintainers in MAINTAINERS.md
```

Comment:
GH is complaining, I think you have to remove empty lines.

Reply:
I believe this is because some of these users have not yet been added as repo admins.

> Unknown owner on line N: make sure @USER exists and has write access to the repository

We will need to grant them write permissions to the repository before merging this PR. But, I was waiting on approval.